rt/frontend/jaxrs/src/main/java/org/apache/cxf/jaxrs/impl/ProvidersImpl.java

```diff
@@ -62,4 +62,8 @@ public ProvidersImpl(Message m) {
     private Type getGenericType(Class<?> type, Type genericType) {
         return genericType == null ? type : genericType;
     }
+
+    public Message getMessage() {
```

Comment:
Probably not a good idea, exposing CXF internals (see please suggestion for `SseMessageBodyReader`)